docs/build/identity.rs/1.1/docs/how-tos/verifiable-credentials/selective-disclosure.mdx

```diff
@@ -0,0 +1,138 @@
+:::
+
+
+#### Validation 🔬
+With these values and a valid signature, the verifier is able to reconstruct a verified VC that exclusively contains the information the holder intended to disclose.
+
+## How it works
+
+As the SD JWT remains a valid JWT, giving the following JWT claim of an address credential in accordance with the [VC Data Model v1.1](https://www.w3.org/TR/vc-data-model/#json-web-token):
+
+
+```json
+{
+  "iss": "did:iota:tst:0x899d07a766f93c2af1a19a3f4583ad338fc94c5d84b6afcadf49b197e1cb693e",
+  "jti": "https://example.com/credentials/3732",
+  "nbf": 1705925652,
+  "sub": "did:iota:tst:0x6c045e1f658197b432cfc7c66350b8781dca50f820e9de0fcdf0029b4b384355",
+  "vc": {
+    "@context": "https://www.w3.org/2018/credentials/v1",
+    "credentialSubject": {
+      "address": {
+        "country": "DE",
+        "locality": "Maxstadt",
+        "postal_code": "12344",
+        "street_address": "Weidenstraße 22"
+      },
+      "name": "Alice"
+    },
+    "type": [
+      "VerifiableCredential",
+      "AddressCredential"
+    ]
+  }
+} 
+
+```
+
+The issuer makes the values of "locality", "postal_code", "street_address" selectively disclosable, giving the holder the freedom to select what details of the address to be disclosed and presented.
+
+```json
+{
+  "_sd_alg": "sha-256",
+  "iss": "did:iota:tst:0x899d07a766f93c2af1a19a3f4583ad338fc94c5d84b6afcadf49b197e1cb693e",
+  "jti": "https://example.com/credentials/3732",
+  "nbf": 1705925652,
+  "sub": "did:iota:tst:0x6c045e1f658197b432cfc7c66350b8781dca50f820e9de0fcdf0029b4b384355",
+  "vc": {
+    "@context": "https://www.w3.org/2018/credentials/v1",
+    "credentialSubject": {
+      "address": {
+        "_sd": [
+          "8Dai0-GMZgkzmdryGzjYufUaRFkiNWzVsJJdWucwu84",
+          "jemTNaG_wiHauwmwWiWREsirAlr91qugPds4MA8e2xo",
+          "iakC9Dfe2r9fGnOaAr_pGg1b7CwITBjcwE7-O7WlMnY"
+        ],
+        "country": "DE"
+      },
+      "name": "Alice"
+    },
+    "type": [
+      "VerifiableCredential",
+      "AddressCredential"
+    ]
+  }
+}
+```
+
+:::note
+The digests are contained in the `_sd` property in `address`. This allows both keys and values to be concealed.
+:::
+
+For further details, see the [example](#full-example-code) below and the [sd-jwt-payload crate](https://github.com/iotaledger/sd-jwt-payload).
+
+## Presentation format
+
+The SD-JWT is presented in the following [format](https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-07.html#section-5):
+
+> `<Issuer-signed JWT>~<Disclosure 1>~<Disclosure 2>~...~<Disclosure N>~<optional KB-JWT>`
+
+## Key Binding JWT
+
+When a verifier receives an SD-JWT, it may be desirable to verify that the identity of the presenter matches the holder of the Credential. For that purpose a [Key Binding JWT (KB-JWT)](https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-07.html#section-4.3) can be used.
```

Comment:
```suggestion
When a verifier receives an SD-JWT, it may be desirable to verify that the presenter's identity matches the holder of the Credential. For that purpose, a [Key Binding JWT (KB-JWT)](https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-07.html#section-4.3) can be used.
```